client/modules/IDE/components/Editor.jsx

```diff
@@ -34,6 +42,16 @@ const unsavedChangesDotUrl = require('../../../images/unsaved-changes-dot.svg');
 const rightArrowUrl = require('../../../images/right-arrow.svg');
 const leftArrowUrl = require('../../../images/left-arrow.svg');
 
+const metaKey = (() => {
```

Comment:
there's also a similar function in `KeyboardShortcuts.jsx`... should this be moved to a utility function?